### DISCUSSION TITLE
useHydrateAtoms console warnings.

### DISCUSSION THREAD

**Discussion by aryankarim** [pmndrs/jotai#2604] [Bug report]
when using useHydrateAtoms, there will be two warnings on the browser's console:

```
Warning: async/await is not yet supported in Client Components.
```
and
```
Warning: A component was suspended by an uncached promise.

**Comment by dai-shi** [comment]
It doesn't sound like related with `useHydrateAtoms`.
Those warning messages can be seen without `useHydrateAtoms`.

**Reply by aryankarim** [reply]
You are right. The `async` on the 'use client' component caused the issue.
